website/docs/docs/introduction.md

```diff
@@ -9,7 +9,38 @@ pagination_prev: null
 
 dbt compiles and runs your analytics code against your data platform, enabling you and your team to collaborate on a single source of truth for metrics, insights, and business definitions. This single source of truth, combined with the ability to define tests for your data, reduces errors when logic changes, and alerts you when issues arise.
 
-Read more about why we want to enable analysts to work more like software engineers in [The dbt Viewpoint](/community/resources/viewpoint).
+Read more about why we want to enable analysts to work more like software engineers in [The dbt Viewpoint](/community/resources/viewpoint). [Join the dbt Community](https://www.getdbt.com/community/join-the-community) to learn how other data practitioners around the world are using dbt, share your own experiences, and get help with your dbt projects.
+
+<Lightbox src="/img/docs/cloud-overview.jpg" width="60%" title="dbt works alongside your ingestion, visualization, and other data tools, so you can transform data directly in your cloud data platform." />
```

Comment:
It might be beneficial for the description here to be a sentence placed above the photo (as a lead in) and the `title` here changed to describe what is in the photo.